recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "eb20b4bf15f20bad149ec82fffac74f16de2a8a797e321a3f8189d63263a511e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cpp-{{ version }}.tar.gz
+  url: https://github.com/msgpack/{{ name }}/archive/cpp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Support latest gtest
+    # https://github.com/msgpack/msgpack-c/commit/4a1db60af8d6bdb258a34352f5cd8d9800544372.patch
+    - gtest.patch
+    # This is needed for a bug in gcc version used where isnan is ambiguous. (::isnan and std::isnan)
+    - ambiguous_nan_inf.patch   # [linux]
+
+build:
+  number: 0
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+  skip: true  # [win]
```

Comment:
More thinking of making it easier for one to switch later. Though agreed this could simply be taken as advisement for the future.